lib/src/main/resources/db/changelog/social-rdbms.db.changelog-1.0.0.xml

```diff
@@ -205,7 +205,7 @@
         <column name="REGISTRATION" type="TINYINT">
           <constraints nullable="false"/>
         </column>
-        <column name="DESCRIPTION" type="LONGTEXT"/>
+        <column name="DESCRIPTION" type="NVARCHAR(1000)"/>
```

Comment:
:+1: 
Please do the test with Postgresql as well.
Then if it is ok I let @nttuyen validate the PR.